pydis_site/context_processors.py

```diff
@@ -1,7 +1,7 @@
 from django.conf import settings
-from django.template import RequestContext
+from django.http import HttpRequest
 
 
-def git_sha_processor(_: RequestContext) -> dict:
+def git_sha_processor(_: HttpRequest) -> dict:
```

Comment:
Hi @GDWR, it's totally fine if you don't remember anymore, but would you happen to know why this change was required in your original PR?

Reply:
its been a while, but iirc the type was incorrect. And just thought it would be nice to correct it incase anybody ever tries to use the param.

Reply:
Oh I see. Thanks!